local/compose/build.go

```diff
@@ -87,7 +86,18 @@ func (s *composeService) Build(ctx context.Context, project *types.Project, opti
 }
 
 func (s *composeService) ensureImagesExists(ctx context.Context, project *types.Project, observedState Containers, quietPull bool) error {
-	images, err := s.getImageDigests(ctx, project)
+	for _, service := range project.Services {
+		if service.Image == "" && service.Build == nil {
+			return fmt.Errorf("invalid service %q. Must specify either image or build", service.Name)
```

Comment:
nit: I think go errors are not supposed to contain caps, what about
```suggestion
			return fmt.Errorf("invalid service %q: must specify either image or build", service.Name)
```
?

Reply:
AFAIk there's a few other places we didn't strictly applied this gopherextremism.
Considering this error was just moved by this PR, to be debated in a separate PR if we want to also apply this "rule"